mopidy_musicbox_webclient/static/js/process_ws.js

```diff
@@ -229,7 +229,7 @@ function processArtistResults (resultArr) {
     resultsToTables(resultArr, ARTIST_TABLE, resultArr.uri)
     var artistname = getArtist(resultArr)
     $('#h_artistname, #artistpopupname').html(artistname)
-    getArtistImage(artistname, '#artistviewimage, #artistpopupimage', 'extralarge')
+    coverArt.getArtistImage(resultArr[0].uri, '#artistviewimage, #artistpopupimage', 'extralarge', mopidy)
```

Comment:
I would have expected to pass the artist (rather than track) uri in here since we have it.